jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -312,6 +317,41 @@ ecma_op_general_object_define_own_property (ecma_object_t *object_p, /**< the ob
   JERRY_ASSERT (property_desc_p->is_enumerable_defined || !property_desc_p->is_enumerable);
   JERRY_ASSERT (property_desc_p->is_writable_defined || !property_desc_p->is_writable);
 
+#ifndef CONFIG_DISABLE_TYPEDARRAY_BUILTIN
```

Comment:
This code should not go here. There is an ecma_op_object_define_own_property, add it there.